frontend/src/components/form/base/EColorField.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+  <EParseField
+    ref="input"
+    :value="value"
+    :format="format"
+    :parse="parse"
+    :serialize="serialize"
+    :deserialize="deserialize"
+    :required="required"
+    :vee-id="veeId"
+    :vee-rules="veeRules"
+    reset-on-blur
+    v-bind="$attrs"
+    v-on="$listeners"
+    @input="$emit('input', $event)"
+  >
+    <!-- passing through all slots -->
+    <slot v-for="(_, name) in $slots" :slot="name" :name="name" />
+    <template v-for="(_, name) in $scopedSlots" :slot="name" slot-scope="slotData">
+      <slot v-if="name !== 'prepend'" :name="name" v-bind="slotData" />
+    </template>
+    <template #prepend="props">
+      <slot name="prepend" v-bind="props">
+        <ColorSwatch
+          class="mt-n1"
+          :color="props.serializedValue"
+          tag="div"
+          :aria-label="props.serializedValue"
+        />
+      </slot>
+    </template>
+  </EParseField>
+</template>
+
+<script>
+import { reactive } from 'vue'
+import { formComponentMixin } from '@/mixins/formComponentMixin.js'
+import { parse, serialize, ColorSpace, sRGB } from 'colorjs.io/fn'
+import ColorSwatch from '@/components/form/base/ColorPicker/ColorSwatch.vue'
+
+export default {
+  name: 'EColorField',
+  components: { ColorSwatch },
+  mixins: [formComponentMixin],
+  props: {
+    value: { type: String, required: false, default: null },
+  },
+  emits: ['input'],
+  setup() {
+    ColorSpace.register(sRGB)
+  },
+  methods: {
+    format(value) {
+      if (typeof value === 'string') {
+        return value
+      }
+      return !value
+        ? ''
+        : serialize(value, { format: 'hex', collapse: false }).toUpperCase()
+    },
+    /**
+     * @param {string} value
+     */
+    parse(value) {
+      if (value === '') {
+        return null
+      }
+      try {
+        const color = parse(value)
```

Comment:
Why do `space` and `format` not have to be specified here? 
Compare with line 96

Reply:
Because we don't know the format / we allow for more formats

Reply:
how do i do that? can you give me an example?

Reply:
I think we should restrict to the sRGB colorspace. But we parse all possible sRGB formats:
- HEX like #FF000
- CSS keyword colors like hotpink
- RGB like rgb(0,1,0)

Reply:
I ask because: 
The call `ColorSpace.register(sRGB)` looks very static. I assume `colorjs.io` behaves differently / more specifically after this call than before. And therefore its behavior depends on whether a ColorPicker has already been displayed or not.

a) Implement behavior locally.  `parse(value, { space: 'srgb' })` ??
b) Move configuration to a more general location

Reply:
Sadly for the procedural access, we need to register the colorspace before we use it. Because of vite I need to register it close to the component, otherwise it does not work. I tried to create a plugin for it, but it failed and the tests did not work anymore.

I do now check if the colorspace has already been registered, before I register it. In that way we don't register it before we need it.

Reply:
I forgot to actually install the plugin. It works now.